data-prepper-plugins/dynamodb-source/README.md

```diff
@@ -52,6 +52,8 @@ source:
 
 * s3_bucket (Required):  The destination bucket to store the exported data files
 * s3_prefix (Optional):  Custom prefix.
+* s3_sse_kms_key_id (Optional):  A AWS KMS Customer Managed Key (CMK) to encrypt the export data files. The key id will
```

Comment:
We have at least three options starting with `s3_`. It may be ideal to make this `s3:` since we haven't yet released this feature.

```
s3:
  bucket: my-bucket
  prefix: /custom/prefix
  sse_kms_key_id: arn:...
```

Reply:
It may be confused this way it seems more options are supported rather than S3. So it's better if we just align this with the API structure: https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_ExportTableToPointInTime.html

Reply:
That makes sense - I'm ok with this.